### PR TITLE
Creating a FFmpeg library along with a test …

### PR DESCRIPTION
**Pull request by fredbertsch**
… that will work whether or not FFmpeg is installed on the machine running the test. This also creates a set of integration tests that will be run during the nightly build.
Change: 119061788

### CODE REVIEW

**Comment by fredbertsch** [comment]
Jenkins, test this please.

**Comment by martinwicke** [comment]
Jenkins, test this please?

File: tensorflow/contrib/ffmpeg/kernels/ffmpeg_lib_test.cc [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =============================================================================
+
+#include "tensorflow/contrib/ffmpeg/kernels/ffmpeg_lib.h"
+
+#include <stdlib.h>
+
+#include "tensorflow/core/lib/core/command_line_flags.h"
```
Comment (vrv) [inline tensorflow/contrib/ffmpeg/kernels/ffmpeg_lib_test.cc]: Hi @fredbertsch: we are trying hard to get rid of command_line_flags from libraries --- can you remove the need for this?

Comment (fredbertsch) [inline tensorflow/contrib/ffmpeg/kernels/ffmpeg_lib_test.cc]: Are we trying to remove them from tests as well? This isn't in a library. For context, this flag is passing in some context about the test environment to the test. We could also do that with an environment variable, but that's even worse.

Comment (vrv) [inline tensorflow/contrib/ffmpeg/kernels/ffmpeg_lib_test.cc]: Trying to get rid of it from everywhere except in example binaries.

**Comment by caisq** [comment]
@tensorflow-jenkins test this please.

**Comment by caisq** [comment]
@tensorflow-jenkins test this please

**Comment by caisq** [comment]
@tensorflow-jenkins test this please

**Comment by caisq** [comment]
@tensorflow-jenkins test this please

**Comment by martinwicke** [comment]
Jenkins, test this please.

**Comment by martinwicke** [comment]
Jenkins, test this please, again.

**Comment by martinwicke** [comment]
Jenkins, test this please

**Comment by caisq** [comment]
@tensorflow-jenkins test this please

**Pull request merged**